app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="text.main.style" parent="TextAppearance.AppCompat.Display1">
+        <item name="android:textStyle">italic</item>
+        <item name="android:fontFamily">monospace</item>
+        <item name="android:textColor">@color/teal_700</item>
+        <item name="android:textSize">@dimen/text_view_text_size_x_big_35</item>
+    </style>
+
+    <style name="button.main.style" parent="TextAppearance.AppCompat.Display1">
+        <item name="android:textStyle">italic</item>
+        <item name="android:textColor">@color/black</item>
+        <item name="android:textSize">@dimen/button_text_size_normal</item>
+    </style>
+</resources>
```

Comment:
to avoid this error you have to add an extra line empty at the end of the file

Reply:
i think this is the AS checkbox to do it automatically
![image](https://user-images.githubusercontent.com/47530344/121538875-861cfe80-c9db-11eb-8c42-57fe4b8496a3.png)

Reply:
Done